app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
```

Comment:
Please look at scaffold controller and takeover most of the code from there or look @casaper controller code from [here](https://github.com/panter/aoz-003/pull/9/files#diff-4e05ad0d64e6100656b63ad1e78f32c5)